R/module_teal.R

```diff
@@ -215,3 +261,234 @@ srv_teal <- function(id, data, modules, filter = teal_slices()) {
 
   invisible(NULL)
 }
+
+
+#' @rdname module_teal_data
+ui_validate_reactive_teal_data <- function(id) {
```

Comment:
Would be easier to review changes in these functions if they stay in the `module_teal_data.R`